advanced/Scripts/chronometer.sh

```diff
@@ -236,7 +236,7 @@ get_sys_stats() {
 
         sys_name=$(hostname)
 
-        [[ -n "$TEMPERATUREUNIT" ]] && temp_unit="$TEMPERATUREUNIT" || temp_unit="c"
+        [[ -n "$TEMPERATUREUNIT" ]] && temp_unit="$TEMPERATUREUNIT" || temp_unit="C"
```

Comment:
Ah, yeah, I see. Could either set the default to `C` or have the `$temp_unit` usages set to `${temp_unit^^}` later in the script so that case doesn't matter. The config line value in `setupVars.conf` can be upper or lower case.

https://github.com/pi-hole/pi-hole/pull/3665/files#diff-3720a64fbbf48da832af105e362ff859R313

and

https://github.com/pi-hole/pi-hole/pull/3665/files#diff-3720a64fbbf48da832af105e362ff859L327


I think the better solution would be to use the uppercase bash variable expansion.

Reply:
I agree it will catch other user input.